src/applications/mhv-secure-messaging/components/Modals/DeleteDraftModal.jsx

```diff
@@ -29,7 +29,7 @@ const DeleteDraftModal = props => {
         <va-button
           id="delete-draft"
           data-testid="confirm-delete-draft"
-          class="vads-u-padding-right--0 vads-u-padding-bottom--1p5 small-screen:vads-u-padding-right--2 small-screen:vads-u-padding-bottom--0"
+          class="vads-u-padding-right--0 vads-u-padding-bottom--1p5 mobile-lg:vads-u-padding-right--2 mobile-lg:vads-u-padding-bottom--0"

```

Comment:
Appears that `mobile-lg` contains padding attriubtes that are being overriden here by `vads-u-padding-right--0`. I can see that removing it should resolve spacing issue as on screenshot 👇🏻 
![image](https://github.com/user-attachments/assets/90dfa987-dc52-43be-80b9-1f6b8e8b33d5)
```suggestion
          class="vads-u-padding-bottom--1p5 mobile-lg:vads-u-padding-right--2 mobile-lg:vads-u-padding-bottom--0"

```